src/System.Data.SqlClient/tests/ManualTests/DataCommon/AssemblyResourceManager.cs

```diff
@@ -55,6 +55,10 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
 
         private bool TryGetResourceValue(string resourceName, object[] args, out object result)
         {
+#if uapaot // Reflection of internal types not supported on UapAot
```

Comment:
I recommend using 
```c#
if (!RuntimeInformation.FrameworkDescription.StartsWith(".NET Native"))
{

} 
else
{

}
```

This avoids creating a new configuration, which keeps things simpler. Since this is just a test, the performance is irrelevant.